<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="b0cf4aef-650d-4e92-b72e-69fad526708f" name="Dan@KARDIA7 2012-12-30 23:55:13" runUser="Kardia7\Dan" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="ローカル" id="16785457-818c-483b-bda1-61626e588443">
    <Description>これらはローカル テスト実行用の既定のテスト設定です。</Description>
    <Deployment enabled="false" runDeploymentRoot="Dan_KARDIA7 2012-12-30 23_55_13">
      <DeploymentItem filename="C:\Users\Dan\Dropbox\CommonLibrary\ファイル屋\bin\Debug\ファイル屋.dll" />
      <DeploymentItem filename="C:\Users\Dan\Dropbox\Demos\NtfsStreams\Trinet.Core.IO.Ntfs\bin\Debug\Trinet.Core.IO.Ntfs.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2012-12-30T23:55:13.8094162+09:00" queuing="2012-12-30T23:55:15.5405152+09:00" start="2012-12-30T23:55:15.6835234+09:00" finish="2012-12-30T23:55:17.4056219+09:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="全ファイル取得Test" storage="c:\users\dan\dropbox\zoneidremover\test_ファイル屋\bin\debug\test_ファイル屋.dll" id="8eee6679-5d1f-6f07-d61c-c765fdaf8167">
      <Execution id="1c0ffa47-115a-4e20-8a97-abd4b894d721" />
      <TestMethod codeBase="C:/Users/Dan/Dropbox/ZoneIDRemover/TEST_ファイル屋/bin/Debug/TEST_ファイル屋.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TEST_ファイル屋.FileExTest, TEST_ファイル屋, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="全ファイル取得Test" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="一覧に存在しない結果" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="読み込まれたすべての結果" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="8eee6679-5d1f-6f07-d61c-c765fdaf8167" executionId="1c0ffa47-115a-4e20-8a97-abd4b894d721" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="1c0ffa47-115a-4e20-8a97-abd4b894d721" testId="8eee6679-5d1f-6f07-d61c-c765fdaf8167" testName="全ファイル取得Test" computerName="KARDIA7" duration="00:00:00.5005488" startTime="2012-12-30T23:55:15.9245372+09:00" endTime="2012-12-30T23:55:17.3466185+09:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="1c0ffa47-115a-4e20-8a97-abd4b894d721">
      <Output>
        <ErrorInfo>
          <Message>テスト メソッド TEST_ファイル屋.FileExTest.全ファイル取得Test が例外をスローしました: 
System.IO.DirectoryNotFoundException: パス 'C:\PathTest' の一部が見つかりませんでした。</Message>
          <StackTrace>    場所 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   場所 System.IO.Directory.Delete(String fullPath, String userPath, Boolean recursive, Boolean checkHost)
   場所 System.IO.Directory.Delete(String path, Boolean recursive)
   場所 TEST_ファイル屋.FileExTest.全ファイル取得Test() 場所 C:\Users\Dan\Dropbox\ZoneIDRemover\TEST_ファイル屋\FileExTest.cs:行 76
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>